ce/custom/tar-stream/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "tar-stream",
```

Comment:
I'm really not keen on checking in this entire separate package including its binary test assets.